src/BSD-1-Clause.xml

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<SPDXLicenseCollection xmlns="http://www.spdx.org/license">
+  <license licenseId="BSD-1-Clause" name="BSD 1-clause License">
```

Comment:
We don't seem to be particularly consistent about this:

```
$ git grep 'name="BSD .*-clause' | wc -l
6
$ git grep 'name="BSD .*-Clause' | wc -l
3
```

but despite its current lower popularity, I prefer `1-Clause`.  But I'm fine leaving this alone until we have an official policy (I've opened #524 to get that policy).